internal/server/proto/server.proto

```diff
@@ -2223,6 +2234,10 @@ message ConfigVar {
   // It exists only to be referenced by other variables.
   bool internal = 8;
 
+  // Indicates that this is actually be written as a file, with the name
+  // field being the filename.
+  bool name_is_path = 9;
```

Comment:
Any reason you didn't use a `oneof` on name instead? It seems like you're effectively making that happen here but using a bool instead feels weird. In my mind I expected something like the below....

```proto
oneof name {
  string env = 1;
  string path = 2
}
```

Reply:
Certainly could! That will ripple out far and wide.

Reply:
Yeah... it would. We can punt it to a future branch experiment.

Reply:
👍